js-core-1/week-2/lesson.md

```diff
@@ -444,20 +457,14 @@ In the node REPL, enter the following array:
 > let fruits = ['banana', 'apple', 'strawberry', 'kiwi', 'fig', 'orange'];
```

Comment:
We should change this to `const` in case it implicitly teaches that you need `let` to mutate an array.

Reply:
I know different people use `const` and `let` in different ways but it's normally either `const` as default or `let` as default. Everywhere else we have `const` as default.